EndlessClient/Dialogs/Factories/ItemTransferDialogFactory.cs

```diff
@@ -44,7 +46,19 @@ public ItemTransferDialog CreateItemTransferDialog(string itemName, ItemTransfer
                 transferType,
                 totalAmount,
                 message);
-            dlg.DialogClosing += (_, _) => _sfxPlayer.PlaySfx(SoundEffectID.DialogButtonClick);
+
+            dlg.DialogClosing += (_, _) =>
+            {
+                if (itemName == "Gold" && !_goldSoundPlayed)
```

Comment:
When you move this logic to the controller, you can add the sound effect to when the "safety comment" message dialog is opened rather than when the item transfer is closed. I think that makes a bit more sense.